### ISSUE TITLE
Change in `options:` key parameters does not clear cache for rebuilds

### ISSUE THREAD

**Issue by AakashGfude** [executablebooks/sphinx-external-toc#46]
When you add/edit/remove a key from the `options` object in `_toc.yml`, the cached objects are still used on subsequent rebuilds. 

The cache should be cleared and a fresh build should happen.

**Comment by AakashGfude** [comment]
Update:

The build is refreshed when you add `numbered: True`.
Ex:
```
- caption: Tutorials
  numbered: True
  chapters:
  - file: start/your-first-book
    sections:
    - file: start/overview
    - file: start/create
    - file: start/build
    - file: start/publish
  - file: tutorials/references
  - file: basics/questions
```
![Screen Shot 2021-07-06 at 12 06 09 pm](https://user-images.githubusercontent.com/6542997/124531794-91d0d900-de52-11eb-954d-6ed61a101746.png)

but when you change it to `numbered: False` it does not refresh
Ex:
```
- caption: Tutorials
  numbered: False
  chapters:
  - file: start/your-first-book
    sections:
    - file: start/overview
    - file: start/create
    - file: start/build
    - file: start/publish
  - file: tutorials/references
  - file: basics/questions
```
**still gives**: 
![Screen Shot 2021-07-06 at 12 06 09 pm](https://user-images.githubusercontent.com/6542997/124531794-91d0d900-de52-11eb-954d-6ed61a101746.png)

**Comment by chrisjsewell** [comment]
hopfully closed in #51, but see my comment on #48

**Comment by chrisjsewell** [comment]
As mentioned, I think this is now fixed, but feel free to re-open if not
